include/tachimawari/control/sdk/module/dynamixel_sdk.hpp

```diff
@@ -70,6 +72,14 @@ class DynamixelSDK : public ControlManager
 
   bool bulk_read_packet() override;
 
+  void bulk_read_proccess(
```

Comment:
You may add this method in its parent class (as a virtual method in `ControlManager` class).